server/src/routes/api/v1/reviewRouter.js

```diff
@@ -3,6 +3,29 @@ import { Review } from "../../../models/index.js";
 
 const reviewRouter = new express.Router();
 
+reviewRouter.patch("/:id", async (req, res) => {
+  const reviewContent = req.body.content;
+
+  const userId = req.body.userId;
+  const reviewId = req.body.reviewId;
+
+  const review = await Review.query().findById(reviewId);
+  const reviewUserId = review.userId;
+
+  if (userId === reviewUserId) {
+    try {
+      const updatedReview = await Review.query().patchAndFetchById(reviewId, {
```

Comment:
consider `$query()` and `.patchAndFetch` as you already queried for the `review` record on line 12, we can use an instance method to update that object, instead of querying for it in the database again when we patch update